test/unit/functionsAsyncAwait.test.js

```diff
@@ -0,0 +1,185 @@
+const {
+  getFilteredsStargazers,
+  getTopFive,
+  sortDataByDate,
+  getSumOfReposStars,
+  getReposMoreThanFiveStars,
+  getLastUpdatedRepos,
+} = require("../../scripts/functionsAsyncAwait.js");
+
+const { data } = require("../fixtures/data.js");
+
+describe("getFilteredsStargazers", () => {
+  test("returns an array with repos with more than 5 stars", () => {
+    expect(getFilteredsStargazers(data)).toEqual([
+      {
+        name: "repo1",
+        stargazers_count: 10,
+        updated_at: "2020-11-18T12:00:00Z",
+      },
+      {
+        name: "repo5",
+        stargazers_count: 9,
+        updated_at: "2024-11-18T12:00:00Z",
+      },
+      {
+        name: "repo6",
+        stargazers_count: 10,
+        updated_at: "2025-11-18T12:00:00Z",
+      },
+      {
+        name: "repo9",
+        stargazers_count: 6,
+        updated_at: "2028-11-18T12:00:00Z",
+      },
+      {
+        name: "repo10",
+        stargazers_count: 8,
+        updated_at: "2029-11-18T12:00:00Z",
+      },
+    ]);
+  });
+
+  test("returns an array with the name and the stars of repos with more than 5 stars", () => {
+    expect(getReposMoreThanFiveStars(data)).toEqual([
+      {
+        name: "repo1",
+        stars: 10,
+      },
+      {
+        name: "repo5",
+        stars: 9,
+      },
+      {
+        name: "repo6",
+        stars: 10,
+      },
+      {
+        name: "repo9",
+        stars: 6,
+      },
+      {
+        name: "repo10",
+        stars: 8,
+      },
+    ]);
+  });
+});
+
+describe("getLastUpdatedRepos", () => {
+  test("returns an array with the top five repos", () => {
```

Comment:
Add another test when the data is an empty array.
```
test("returns an array with the top five repos when data array is filled with info", () => {
test("returns an empty array with data array is empty", () => {
test("returns an empty array with data array is null", () => {
test("returns an array with the repos when data array is filled with just 3 items", () => {
```
Do this for all the functions